testing/android_background_image/android/build.gradle

```diff
@@ -6,7 +6,7 @@ buildscript {
         mavenCentral()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:7.0.0'
+        classpath 'com.android.tools.build:gradle:8.0.1'
```

Comment:
why is this inconsistent with the one above?

Reply:
These were both the minimal upgrades I could make to the AGP versions in order for builds to succeed. I can try bumping the other one to 8 as well. 

I was forced to upgrade to 8 here because 7->7.3 enabled a new lint, and when I looked into it I found that the lint had been disabled in a newer version because it was unhelpful (hence the 7.3->8.0.1 jump)

Reply:
A lint specific to this test app, that is

Reply:
I bumped the other one as well